docs/changelog-fragments.d/454.misc.md

```diff
@@ -0,0 +1,4 @@
+Added changelog fragment management infrastructure using [Towncrier]
```

Comment:
This is a changelog fragment like the PR authors would be required to add. Its content should be markdown, its name should start with a PR or an issue number, followed by a type specified in `pyproject.toml`, then there's an optional part number for cases when you want to have multiple entries per issue/PR or the same type, and then there's an optional `.md` extension (it will be mandatory with the GH App config, if activated).